### ISSUE TITLE
Update mock-res

### ISSUE THREAD

**Issue by Dallas62** [balderdashy/sails#2708]
Hi!

`mock-res` has been updated to `0.2.1` and implement `writeHead`,
Is https://github.com/balderdashy/sails/blob/master/lib/router/res.js#L96 still needed?

refer to: https://github.com/diachedelic/mock-res/commit/9b759ab2c0a18500cab1c076ad37bd721b3b08eb

PS: Pull Request for mock-req done: https://github.com/diachedelic/mock-req/pull/2 (https://github.com/balderdashy/sails/blob/master/lib/router/req.js#L35)

**Comment by Dallas62** [comment]
https://www.npmjs.com/package/mock-req
Has been update to 0.2.0 and include a fix for
https://github.com/balderdashy/sails/blob/master/lib/router/req.js#L35

**Comment by ConnerMan** [comment]
Hey, trying to help out :) 

I took a look at both of these issues and found the following: 

- Updating mock-req to 0.2.0, does seem to remove the need for https://github.com/balderdashy/sails/blob/master/lib/router/req.js#L35 All tests pass :white_check_mark: 
- Updating to mock-res 0.2.1 and using its writeHead implmentation instead of  (https://github.com/balderdashy/sails/blob/master/lib/router/res.js#L96) causes test failures. :x: 

Is it worth submitting a PR just for the mock-req upgrade?

**Comment by loicsaintroch** [comment]
Ping @mikermcneil and @sgress454

**Comment by sailsbot** [comment]
Thanks for posting, @Dallas62.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
